content/30.conclusions.md

```diff
@@ -1,8 +1,9 @@
 ## Conclusions
 
-The presence of female STEM role models is associated with higher persistence for undergraduate women in geoscience [@doi:10.1130/GES01659.1].
+The underrepresentation of women as corresponding authors observed here is also pervasive in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956].
```

Comment:
```suggestion
We estimate the composition of the field using corresponding author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented among such authors.
Such an effect would cause us to underestimate the number of women in the field.
```
It seems hard to say underrepresentation here because we're using corresponding author position to assess the field. What about this framing?

Reply:
Let me think about this a bit more.